samples/model-builder/test_constants.py

```diff
@@ -197,3 +197,12 @@
 )
 PYTHON_MODULE_NAME = "trainer.task"
 MODEL_TYPE = "CLOUD"
+
+# Feature store constants
+FEAUTURESTORE_NAME = "featurestore_sample"
+ENTITY_TYPE_ID = "entity_type_sample"
```

Comment:
can have both `*_ID`, and `*_NAME` for entity_type and feature, as recommended in the previous PR.

Reply:
added new constants